code/script.js

```diff
@@ -0,0 +1,136 @@
+            const forkedRepos =
+                data.filter(repo => {
+                    return repo.fork && repo.name.startsWith("project-") && selectedLanguages.includes(repo.language)
+                })
+            //Function to sort repos by latest date
+            forkedRepos.sort(function (a, b) {
+                return new Date(b.pushed_at) - new Date(a.pushed_at)
+            })
+            forkedRepos.forEach(repo => {
+                projectsContainer.innerHTML += `
+                      <div class="repo" id=${repo.name}>
+                        <p>Name: ${repo.name} </p> 
+                        <a href="${repo.html_url}" target="blank">Go to repo!</a> 
+                        <p>Branch: ${repo.default_branch}</p>
+                        <p>Latest push: ${new Date(repo.pushed_at).toDateString()}</p>
+                        <p id="commit-${repo.name}">Amount of commits: </p> 
+                      </div>
+                        `
+            })
+
+            fetchPullRequests(forkedRepos)
+
+            //Only draw the chart once
+            if (!chartDrawn) {
+                drawChart(forkedRepos.length)
+                chartDrawn = true
+            }
+        })
+}
+
+const fetchPullRequests = (repos) => {
+    repos.forEach(repo => {
+        const PULL_URL = `https://api.github.com/repos/Technigo/${repo.name}/pulls?per_page=100`
+
+        fetch(PULL_URL)
+            .then(res => res.json())
+            .then((data) => {
+                const myPullRequest =
+                    data.find(pull => pull.user.login === repo.owner.login)
+                if (myPullRequest) {
+                    fetchCommits(myPullRequest.commits_url, repo.name);
+                } else {
+                    document.getElementById(`commit-${repo.name}`).innerHTML =
+                        "No pull request yet!";
+                }
+            })
+    })
+}
+
+const fetchCommits = (myCommitsUrl, myRepoName) => {
+    fetch(myCommitsUrl)
+        .then(res => res.json())
+        .then(data => {
+            document.getElementById(`commit-${myRepoName}`).innerHTML += data.length
+        })
+}
+
+const filterLanguages = (event) => {
+    console.log(event)
+    const language = event.target.id
+    const checked = event.target.checked
+
+    console.log(language, checked)
+
+    //Adds language string in array if checked language is missing
+    if (checked && !selectedLanguages.includes(language)) {
+        selectedLanguages.push(language)
+        //Removes language string in array if unchecked language is in array     
+    } else if (!checked && selectedLanguages.includes(language)) {
+        for (let i = 0; i < selectedLanguages.length; i++) {
+            console.log("i=", i)
+            console.log("selectedLanguages[i]=", selectedLanguages[i])
+            if (selectedLanguages[i] === language) {
+                selectedLanguages.splice(i, 1)
+                break
```

Comment:
Interesting use of break , I had seen it within switch context only . Thank you for educational tip . I had googled and yep thats one of ways (https://stackoverflow.com/questions/4851657/call-break-in-nested-if-statements).